airflow-core/src/airflow/api_fastapi/core_api/services/public/connections.py

```diff
@@ -34,6 +34,26 @@
 from airflow.models.connection import Connection
 
 
+def update_orm_from_pydantic(
+    orm_conn: Connection, pydantic_conn: ConnectionBody, update_mask: list[str] | None = None
+):
```

Comment:
return type annotation

Reply:
Okay, return type is `None` - usually we do not define it. But okay, adjusted.